alphagov/govwifi-admin
# Enhanced 2FA functionality

## What is it?

Replace the current 2FA setup screen with a couple of options:

1. Email 2FA (new)
1. Authentication app (existing)

## Why?

We are going to be rolling out email-based 2FA functionality quite soon. This is the first part of the puzzle.

## How?

This functionality has been implemented in way so that it can be enabled using the feature flag `enable_enhanced_2fa_experience`, which has been defined as a configuration key.

---

### How to test?

Run it locally and take it for a spin.

The email 2FA journey is complete, but will not actually send an actual email (sending email will be implemented in the next phase).

The app 2FA journey should work how it already does.
